client/src/components/Users.js

```diff
@@ -152,7 +152,9 @@ function Users(props) {
     }
   }
 
-  React.useEffect(load, [])
```

Comment:
Was this so bad?

Reply:
No, sorry. I meant to change this back.

Reply:
Scratch that, it does cause a problem. If the function is passed directly as a parameter like this, it causes a runtime warning in the browser:

```
Warning: An effect function must not return anything besides a function, which is used for clean-up.

It looks like you wrote useEffect(async () => ...) or returned a Promise. Instead, write the async function inside your effect and call it immediately:

useEffect(() => {
  async function fetchData() {
    // You can await here
    const response = await MyAPI.getData(someId);
    // ...
  }
  fetchData();
}, [someId]); // Or [] if effect doesn't need props or state
```

When you navigate away from the User Manager page, React then fails to clean up correctly and throws some nasty errors.

Ideally, the async function would sit inside the scope of the `useEffect` callback, but in this instance it is called in several other places and I wanted to avoid too much rework.